distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/XSearch.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.itests.common;
+
+import java.io.ByteArrayInputStream;
+import java.nio.charset.StandardCharsets;
+
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.xpath.XPath;
+import javax.xml.xpath.XPathExpression;
+import javax.xml.xpath.XPathFactory;
+
+import org.w3c.dom.Document;
+
+/**
+ * Simpler helper class for XPath and DOM related functions.
+ * The purpose is to reduce the amount of duplicated code used in the test classes.
+ * <p>
+ * This class is not thread-safe. It not optimized-- it creates new factories and
+ * builders for every invocation. It is not suitable for use in production code.
+ * <p>
+ * This class does not catch errors. It passes errors up the call chain to provide developers with
+ * the most information possible when debugging test cases.
+ * <p>
+ * Unlike ddf.util.XPathHelper, this class can ignore XML namespaces.
+ * <p>
+ * To format (pretty print) XML, use the ddf.util.XPathHelper or
+ * org.codice.ddf.platform.util.XMLUtils classes.
+ * <p>
+ * For XML related test assertions, use an org.custommonkey.xmlunit.XMLAssert methods like
+ * assertXpathEvaluatesTo().
+ * <p>
+ * For XML related test assertions in REST Assured responses,
+ * use the Hamcrest matcher method hasXpath().
+ */
+public class XSearch {
```

Comment:
❗️ The name makes it difficult to know what should go in this class. Do you think we should consider breaking this up into two classes, one for DOM and one for XPath related methods?

Maybe have a `DocumentFactory` with the first two `getDocument()` methods in it, and a second class, called something like `XPathExpression`? That second class could be an `javax.xml.xpath.XPathExpression` decorator with a `static` `compile()` method to create a new instance of the decorator and a the same `evaluate(String)` method you added on top of the basic `XPathExpression` methods.